styleguide/tests/backstop/backstop_data/html_report/bower.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "BackstopJS_compare",
+  "version": "0.0.0",
+  "main": "index.html",
+  "ignore": [
+    "node_modules"
+  ],
+  "dependencies": {
+    "angular": "~1.4.8",
+    "bootstrap-css-only": "~3.3.6",
+    "angular-bootstrap": "~2.1.1",
+    "angular-clipboard": "~1.5.0"
```

Comment:
What's all this here for?  (Angular, bootstrap).  If it's just for testing, document that in the PR